README.md

```diff
@@ -1,57 +1,59 @@
-JavaScript frameworks. You can use it in a browser or as a service with
-Node.js as your back-end technology.
-
-The engine comes with a built-in, localized
-[library of functions](https://handsontable.github.io/hyperformula/guide/built-in-functions.html)
-grouped into different categories, and covers most user-triggered actions
-such as [CRUD operations](https://handsontable.github.io/hyperformula/guide/basic-operations.html),
-[undo/redo](https://handsontable.github.io/hyperformula/guide/undo-redo.html),
-and [clipboard operations](https://handsontable.github.io/hyperformula/guide/clipboard-operations.html).
-It also supports the use of cross-sheet references,
-[named expressions](https://handsontable.github.io/hyperformula/guide/named-expressions.html),
-different [data types](https://handsontable.github.io/hyperformula/guide/types-of-values.html),
-and [custom functions](https://handsontable.github.io/hyperformula/guide/custom-functions.html).
+<br>
+<p align="center">
+  <a href="https://handsontable.github.io/hyperformula/">
+    <img src="https://raw.githubusercontent.com/handsontable/hyperformula/master/github-hf-logo-blue.svg" width="350" height="71" alt="HyperFormula - An advanced headless evaluator and parser of the Excel formulas"/>
+  </a>
+</p>
+
+<p align="center">
+  <strong>An advanced headless evaluator and parser of the Excel formulas</strong>
+</p>
+
+<p align="center">
+  <a href="https://github.com/handsontable/hyperformula/actions?query=workflow%3ATest+branch%3Amaster"><img src="https://img.shields.io/github/workflow/status/handsontable/hyperformula/Test" alt="GitHub Workflow Status"></a>
+  <a href="https://app.fossa.io/projects/git%2Bgithub.com%2Fhandsontable%2Fhyperformula?ref=badge_shield"><img src="https://app.fossa.io/api/projects/git%2Bgithub.com%2Fhandsontable%2Fhyperformula.svg?type=shield" alt="FOSSA Status"></a>
+  <a href="https://lgtm.com/projects/g/handsontable/hyperformula/context:javascript"><img src="https://img.shields.io/lgtm/grade/javascript/g/handsontable/hyperformula.svg?logo=lgtm&amp;logoWidth=18" alt="Language grade: JavaScript"></a>
+  <a href="https://snyk.io/test/github/handsontable/hyperformula?targetFile=package.json"><img src="https://snyk.io/test/github/handsontable/hyperformula/badge.svg?targetFile=package.json" alt="Known Vulnerabilities"></a>
+  <a href="https://github.com/handsontable/hyperformula/graphs/contributors"><img src="https://img.shields.io/github/contributors/handsontable/hyperformula" alt="GitHub contributors"></a>
+  <a href="https://codecov.io/gh/handsontable/hyperformula"><img src="https://codecov.io/gh/handsontable/hyperformula/branch/master/graph/badge.svg?token=5k9ZQv8azO" alt="codecov"></a>
+</p>
+
+---
+
+HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as an evaluator and parser for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
```

Comment:
```suggestion
HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as an evaluator and parser of Excel formulas for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
```

Reply:
I changed it to:
```
HyperFormula is a headless spreadsheet built on top of TypeScript. It serves as a parser and evaluator of Excel formulas for web applications. You can use it in a browser or as a service, with Node.js as your back-end technology.
```
Writing parser and evaluator (in this order) makes more sense because you always parse a formula first before evaluating it.

Reply:
@krzysztofspilka agrees